README.md

```diff
@@ -14,6 +14,8 @@ To install the `IPython Clusters` tab in Jupyter Notebook, add this to your `jup
 c.NotebookApp.server_extensions.append('ipyparallel.nbextension')
 ```
 
+See the [documentation on configuring the notebook server](http://jupyter-notebook.readthedocs.org/en/latest/examples/Notebook/Configuring%20the%20Notebook%20and%20Server.html)
```

Comment:
Guh! I just went straight from what was in my URL bar. Didn't even know https was an option. I'd rather that be the link anyway, so the HTTPS ones are the defaults that people get to.